docs/events/usage-examples/running-tests-with-helpers.md

```diff
@@ -0,0 +1,107 @@
+# Running tests with helpers {#running-tests-with-helpers}
+
+Let's look at the [implementation][testplane-passive-browsers-index] of the [testplane-passive-browsers][testplane-passive-browsers] plugin.
```

Comment:
Hmm, this plugin is deprecated. Maybe we should reflect that somehow.

Reply:
Done